itl/command-plugins.conf

```diff
@@ -1696,6 +1696,7 @@ object CheckCommand "by_ssh" {
 	}
 
 	vars.by_ssh_address = "$check_address$"
+	vars.by_ssh_port = "$ssh_port$"
```

Comment:
The case where I came across this is:
I have a host that I can only check via a reverse SSH tunnel, so I have to use `check_command = "ssh"` on the host level. To make that work, I have to set `vars.ssh_port = ####`. My reasoning was that redundant configuration in this case is not desirable.

But, to be honest, I didn't think about the normal case where vars.ssh_port is **not** set on the host level.